lib/libutils/ext/include/util.h

```diff
@@ -188,7 +188,40 @@ static inline void reg_pair_from_64(uint64_t val, uint32_t *reg0,
 	*reg1 = low32_from_64(val);
 }
 
-/* Get and set bit fields  */
+/*
+ * Functions to get and set bit fields in a 32/64-bit bitfield.
+ *
+ * These helper functions allow setting and extracting specific bits in
+ * a bitfield @reg according to a given mask @mask. The mask
+ * specifies which bits in the bitfield should be updated or extracted.
+ * These functions exist in both 32-bit and 64-bit versions
+ *
+ * set_field_u32()
+ * set_field_u64() - Modifies specific bits in a bitfield by clearing
+ * the bits specified by the mask and then setting these bits to the new
+ * value @val.
```

Comment:
Indent so that it is readable. I think a single line description is simpler :(
```c
 * set_field_u32()
 * set_field_u64() - Modifies specific bits in a bitfield by clearing the bits
 *                   specified by the mask and then setting these bits to the
 *                   new value @val.

Reply:
OK for indentation. The single line description would make the text harder to read in my view.